frontend/src/app/patients/AddPatient.tsx

```diff
@@ -350,7 +354,7 @@ const AddPatient = () => {
                     <use xlinkHref={iconSprite + "#arrow_back"}></use>
                   </svg>
                   <LinkWithQuery to={`/patients`} className="margin-left-05">
-                    People
+                    {PATIENT_TERM_PLURAL_CAP}
```

Comment:
Why are _CAP variations created as variables instead of using the [toUppercase](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase ) method to modify the lower variations?